docs/README.md

```diff
@@ -90,7 +90,7 @@ This will be replaced with the value of the variable called `NAME` when generati
 
 #### Headings
 
-All documents are structured with headings. From these headings, you can automatically generate a page table of contents (see below).
+All documents are structured with headings, written in "Title Case". From these headings, you can automatically generate a page table of contents (see below).
```

Comment:
It seems that this could be left as an implicit assumption for "headings".

Reply:
@greghogan Not sure, I have found a lot of inconsistent headings in the docs, so it seemed worthwhile adding, doesn't mean anyone will follow it of course :)